tox.ini

```diff
@@ -53,9 +53,13 @@ show-source = True
 # N802 skipped (function name should be lowercase)
 # N806 skipped (variable in function should be lowercase)
 # F401 skipped (imported but unused) after verifying current usage is valid
-# W503 skipped line break before binary operator
+# W503 skipped (line break before binary operator)
+# W504 skipped (line break after binary operator)
+# E501 skipped (maximum line length)
```

Comment:
Do we want to skip the max line length test altogether, or increase the max allowable line?

`max-line-length = 99`

Reply:
I skipped this because there were ~35 long lines to tidy, but will increase the limit instead. Should eventually fix all of these, but was aiming to tidy up enough for a release.

Reply:
Cool. The default is 79 for historical reasons, but personally I like 99. Not too many people looking at code on displays limited to 80 chars these days, I think.